mod/01_rating/set/type/metric_value.rb

```diff
@@ -94,13 +102,13 @@ def create_source
   def default_new_args args
     if !args[:source]
       args[:hidden] = {
-        :success=>{:id=>'_self', :soft_redirect=>true, :view=>:titled},
+        :success => { id: '_self', soft_redirect: true, view: :titled },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.